CONTRIBUTING.md

```diff
@@ -0,0 +1,168 @@
+Flight Control Contributor Policy
+
+Welcome!
+Thank you for considering a contribution to Flight Control. We appreciate your time and effort in helping us improve this project. This document explains how to contribute effectively and responsibly, following CNCF Best Practices and the Contributor Covenant v2.1 to maintain a welcoming, open source community.
+1. Before You Get Started
+
+    GitHub Account
+
+        You’ll need a GitHub account with a verified email address.
+        - How to verify: https://docs.github.com/en/account-and-profile/email-visibility/about-email-verification
+
+    Developer Certificate of Origin (DCO)
+
+        All commits must be “signed off” using the -s flag to confirm you agree with the DCO.
+
+        git commit -s -m "Add new feature"
+
+        This ensures you have the right to submit the code and that you agree to release it under the project’s open source license.
```

Comment:
We're also currently asking commit to be signed. The simplest way would be users adding their SSH public key once more to GitHub but with type "signing" (see https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account), then configure their local git to use this as signing key (see https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key#telling-git-about-your-ssh-key).

Then all commits should be signed with `git commit -S -s -m "Add new features".

Reply:
Oh, we also currently require commits to be prefixed with a Jira issue number currently. Contributors who do not have a Jira issue number should prefix "NO-ISSUE:", e.g. "NO-ISSUE: My PR title"